src/java.desktop/share/classes/sun/font/CodePointIterator.java

```diff
@@ -56,15 +56,13 @@ public static CodePointIterator create(char[] text, int start, int limit) {
         return new CharArrayCodePointIterator(text, start, limit);
     }
 
-    public static CodePointIterator create(CharSequence text) {
-        return new CharSequenceCodePointIterator(text);
-    }
-
     public static CodePointIterator create(CharacterIterator iter) {
         return new CharacterIteratorCodePointIterator(iter);
     }
+
 }
 
+
```

Comment:
looks like a couple of unnecessary newlines were added inadvertently

Reply:
I had already noticed that and removed them